src/threading/thread.cpp

```diff
@@ -311,7 +311,7 @@ bool Thread::bindToProcessor(unsigned int proc_number)
 
 bool Thread::setPriority(int prio)
 {
-#if USE_WIN_THREADS
+#ifdef _WIN32
```

Comment:
this is absolutely not how this should be done

change the header file at the point where decision making is done